scripts/rpm-local-build.sh

```diff
@@ -1,18 +1,12 @@
 #!/usr/bin/env bash
 set -e
+set +x
 
 if [[ ! -d dist/rpmbuild  ]]; then
 	echo "Cannot build as artifacts are not generated. Run scrips/rpm-prepare.sh first"
 	exit 1
 fi
 
-echo "Cleaning up old rpmcontent"
-rm -f ~/rpmbuild/{BUILD,RPMS,SOURCES,SPECS,SRPMS}
-mkdir -p ~/rpmbuild/{BUILD,RPMS,SOURCES,SPECS,SRPMS}
-
-echo "Copying content to local rpmbuild"
-cp -avrf dist/rpmbuild/SOURCES/* $HOME/rpmbuild/SOURCES/
-cp -avrf dist/rpmbuild/SPECS/* $HOME/rpmbuild/SPECS/
-
+top_dir="`pwd`/dist/rpmbuild"
 echo "Building locally"
-rpmbuild -ba $HOME/rpmbuild/SPECS/openshift-odo.spec
+rpmbuild --define "_topdir `echo $top_dir`" -ba dist/rpmbuild/SPECS/openshift-odo.spec
```

Comment:
why not use `$top_dir` in `dist/rpmbuild/SPECS/openshift-odo.spec`?

Reply:
We only want to use dist as topdir for local runs, not when we are building for release